src/pages/developers/tutorials.tsx

```diff
@@ -428,7 +425,7 @@ const TutorialsPage = ({
                 {/* TODO: Refactor each tutorial tag as a component */}
                 <Emoji text=":writing_hand:" fontSize="sm" mr={2} />
                 {tutorial.author} •
-                {published(intl.locale, tutorial.published ?? "")}
+                {published(language, tutorial.published ?? "")}
```

Comment:
Not something to fix here, but noting we should require tutorials to have published dates on build.